.github/workflows/Kogito-Operator-PR-Check.yaml

```diff
@@ -72,6 +72,27 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
+      - name: Setup Go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+        id: go
+      - name: Cache dependencies
```

Comment:
We don't need this since it's just a binary install

Reply:
removed